src/store/movies-store/deleteMoviesById.thunk.ts

```diff
@@ -0,0 +1,19 @@
+import {moviesSlice} from "./movies.slice";
+import {moviesService} from "../../api/movies-api/movies.service";
+import {Movie} from "../../api/movies-api/models/movie.model";
+
+export const deleteMoviesByIdThunk = (movieId: Movie) => {
+  return async (dispatch: any) => {
+    dispatch(moviesSlice.actions.deleteMoviesById(movieId))
```

Comment:
I would change `delete` methods on 7 and 10 lines to avoid confusing. If I am not wrong I've already left similar comment in scope of the previous PR

Reply:
I don't think it makes sense